libraries/ms-helper/test/test_can_interval.c

```diff
@@ -20,7 +20,7 @@
 #define TEST_CAN_INTERVAL_FAULT 1
 #define TEST_CAN_INTERVAL_SEND_DELAY_US 100000
 #define TEST_CAN_INTERVAL_PERIOD_US 1000000
-#define NUM_GENERIC_CAN_RX_HANDLERS 5
+#define NUM_GENERIC_CAN_RX_HANDLERS 12
```

Comment:
Why is this defined here?

Reply:
@ckitagawa had it there, but I believe technically, any other value would cause a macro redefined warning..

Reply:
They were probably just missed and should be removed.